src/js/views/eva-variant-view.js

```diff
@@ -463,7 +481,7 @@ EvaVariantView.prototype = {
         if (this.variantIsDeprecated) {return}
 
         this.variant.forEach(function(variantObjFromAccService) {
-            if (_this.accessionCategory === "clustered-variants") {
+            if (_this.accessionCategory === "clustered-variants" && !_this.summaryOnly) {
```

Comment:
ugh.

I hate those little checks that need to be added here and there. They are suggestions that there is something wrong with the design

Reply:
It is wasteful to get associated SS IDs for an RS ID for a multi-variant view since we are only displaying the summary. I agree that this is a bit ugly from a design standpoint if all of this is going to be thrown away in a few months after the website revamp I feel that there is little sense in refactoring.